palantir-java-format/src/main/java/com/palantir/javaformat/java/Main.java

```diff
@@ -215,10 +220,10 @@ public static CommandLineOptions processArgs(String... args) throws UsageExcepti
         try {
             parameters = CommandLineOptionsParser.parse(Arrays.asList(args));
         } catch (IllegalArgumentException e) {
-            throw new UsageException(e.getMessage());
+            throw new UsageException(Optional.ofNullable(e.getMessage()).orElse("null"));
```

Comment:
I don't think we need to change this, but in other types of hot paths [`java.until.Objects.requireNonNullElse`](https://docs.oracle.com/en%2Fjava%2Fjavase%2F11%2Fdocs%2Fapi%2F%2F/java.base/java/util/Objects.html#requireNonNullElse(T,T)) can avoid the Optional allocation

```suggestion
            throw new UsageException(Objects. requireNonNullElse(e.getMessage(), "null"));
```

Reply:
Thanks for sharing -- I've needed this before and its good to know a way to do this without the allocation